SETUP.md

```diff
@@ -230,16 +230,20 @@ docker exec -it local.spark-master /bin/bash
 ```
 
 Start `spark-shell` with the following command: Available users are `openhouse` and `u_tableowner`.
+
 ```
-bin/spark-shell --packages org.apache.iceberg:iceberg-spark-runtime-3.1_2.12:1.2.0   \
-  --jars openhouse-spark-runtime_2.12-*-all.jar  \
+bin/spark-shell --packages org.apache.iceberg:iceberg-azure:1.5.0,org.apache.iceberg:iceberg-spark-runtime-3.1_2.12:1.2.0 \
```

Comment:
So we still need to pass Iceberg-azure as part of packages instead of dependency on spark-runtime as OH currently supports iceberg v1.2?

Reply:
I think so because version 1.2 doesn't have the iceberg-azure implementation

Reply:
Sort of, there are also some jar dependency issues that I had to change that also allowed for it.